src/api/Metadata.js

```diff
@@ -285,20 +285,19 @@ class Metadata extends File {
     /**
      * Gets classification for a file.
      *
-     * @param {BoxItem} file - File object for which we are getting classification
+     * @param {string} id - The file id
      * @param {Function} successCallback - Success callback
      * @param {Function} errorCallback - Error callback
      * @param {boolean|void} [options.forceFetch] - Optionally Bypasses the cache
      * @param {boolean|void} [options.refreshCache] - Optionally Updates the cache
      * @return {Promise}
      */
     async getClassification(
-        file: BoxItem,
+        id: string,
```

Comment:
Can we name this `fileId`? Otherwise it looks like we're "getting a classification with ID of X".

Reply:
Yeah I can, was just trying to do minimal changes to reduce the diff